app/models/ability.rb

```diff
@@ -2,80 +2,75 @@ class Ability
-        { :coach_id => user.id } }
-      can :read, Feedback, { :submitted => true, :user_id => user.id }
-      can :read, Feedback, { :submitted => true, :review => { :associate_consultant => { :user_id => user.id } } }
-      can :read, Feedback, { :submitted => true, :review => { :associate_consultant => { :coach_id => user.id } } }
-      can :read, Feedback, { :submitted => true, :review => { :associate_consultant => { :reviewing_group_id => user.reviewing_group_ids } } }
-
-      can [:summary, :read], Review, :associate_consultant => { :user_id => user.id }
-      can [:summary, :read, :coachees], Review, :associate_consultant => { :coach_id => user.id }
-      can [:summary, :read], Review, :associate_consultant => { :reviewing_group_id => user.reviewing_group_ids }
-      can [:update], Review do |review|
-        review.associate_consultant.user == user && review.review_date > Date.today
-      end
-
-      can [:show, :update, :feedbacks, :completed_feedback, :add_email, :remove_email], User, :id => user.id
-
-      # admin permissions
-      if user.admin
-        can :manage, Review
-        can :manage, ReviewingGroup
-        can :manage, AssociateConsultant
-        can :manage, User
-        can :read, Invitation do |invitation|
-          invitation.review.upcoming?
-        end
-        can [:summary, :index, :read], Feedback, { submitted: true }
-      end
+    can :manage, [Review, ReviewingGroup, AssociateConsultant, User]  if user.admin?
+
+    # User
+    can(:create, User) if user.new_record?
+    can [:show, :update, :feedbacks, :completed_feedback, :add_email, :remove_email], User, :id => user.id
+
+    # Invitation
+    can :manage, Invitation do | invitation |
+      invitation.review.upcoming? && review_participant?(invitation.review, user)
+    end
+
+    can(:read, Invitation) { |invitation| invitation.review.upcoming? } if user.admin?
+
+    can [:read, :destroy], Invitation, email: user.email
+    can [:read, :destroy], Invitation do |invite|
+      sent_to_alias?(invite.review, user)
     end
+
+    # Feedback
+    can [:read, :update, :destroy], Feedback, submitted: false, user_id: user.id
+    can [:summary, :index, :read],  Feedback, { submitted: true } if user.admin?
+
+    can [:create, :new], Feedback do | feedback |
+      feedback.review.upcoming? && (
+      !feedback.review.invitations.where(email: user.email).empty? ||
+      own_review?(feedback.review, user))
+    end
+
+    can :send_reminder, Feedback, :review => { :associate_consultant => { :user_id => user.id } }
+    can :send_reminder, Feedback, :review => { :associate_consultant => { :coach_id => user.id } }
+    can :read, Feedback, { :submitted => true, :user_id => user.id }
+    can :read, Feedback, { :submitted => true, :review => { :associate_consultant => { :user_id => user.id } } }
+    can :read, Feedback, { :submitted => true, :review => { :associate_consultant => { :coach_id => user.id } } }
```

Comment:
Line is too long. [113/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.